src/Mono.Android/Android.Runtime/JNIEnv.cs

```diff
@@ -296,12 +294,8 @@ internal static void PropagateUncaughtException (IntPtr env, IntPtr javaThreadPt
 		[DllImport ("__Internal", CallingConvention = CallingConvention.Cdecl)]
 		extern static void _monodroid_gc_wait_for_bridge_processing ();
 
-		static volatile bool BridgeProcessing; // = false
```

Comment:
Should not be removed: https://github.com/xamarin/xamarin-android/blob/0ddcc3fbd8458a4749753b08dd9a081c1e6036c4/src/monodroid/jni/monodroid-glue.cc#L982